abs.yaml

```diff
@@ -0,0 +1,10 @@
+upload_channels:
+  - sfe1ed40
+
+build_parameters:
+  - "--no-test"
+
```

Comment:
```suggestion
```
Since you actually don't need the dep on langgraph, this can be removed. 

If we did want to set --no-test, we would want to keep the default options in the list: https://github.com/anaconda-distribution/rocket-platform/blob/main/cookbook/flows/config.yaml#L19-L22